helm/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 
 This file documents all notable changes to [Gravitee.io API Management 3.x](https://github.com/gravitee-io/helm-charts/tree/master/apim/3.x) Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
 
+### 4.0.1
+- 
+- Inherit notifiers.smtp from root smtp configuration
```

Comment:
Should we highlight that it is a breaking change?

Reply:
I have to add this change on the 4.0.0 too?